File: Curso Android - Aprenda a criar 15 apps/ProjetoFragmento/app/src/main/java/projetofragmento/cursoandroid/com/projetofragmento/MainActivity.java
package projetofragmento.cursoandroid.com.projetofragmento;

import android.app.FragmentManager;
import android.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.RelativeLayout;

public class MainActivity extends AppCompatActivity {
    private Button button;
    private boolean status = true;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        button = (Button)findViewById(R.id.button);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                android.support.v4.app.FragmentManager fragmentManager = getSupportFragmentManager();
                android.support.v4.app.FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
                if(status){
                    LoginFragment loginFragment = new LoginFragment();
                    fragmentTransaction.add(R.id.relative,loginFragment);
                    fragmentTransaction.commit();
                    button.setText("Cadastre-se");
                    status = false;
                }else
                {
                    CadastroFragment cadastroFragmentt = new CadastroFragment();
                    fragmentTransaction.add(R.id.relative,cadastroFragmentt);
                    fragmentTransaction.commit();
                    button.setText("Logar");
                    status = true;
                }
            }
        });
    }
}

File: Curso completo de Kotlin/PrimeiroprojetoCurso/app/src/main/java/com/cursoandroidkotlin/isaque/primeiroprojetocurso/MainActivity.kt
package com.cursoandroidkotlin.isaque.primeiroprojetocurso

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
    fun Jogar(view: View)
    {
        var texto = findViewById<TextView>(R.id.textosorteio)
        // var texto = findViewById(R.id.textosorteio) as TextView
        var numeroSorteado = Random.nextInt(11)
        texto.setText("Numéro sorteado é: $numeroSorteado")
    }
}
